### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by addyosmani** [raDiesle/backbone.js-jquerymobile-boilerplate-template#7]
Hi there. I just came across this project and was wondering if you were interested in continuing working on it. It's a great start for something that could end up in [Backbone Fundamentals](http://backbonefundamentals.com). 

I'm currently [looking](https://github.com/addyosmani/backbone-fundamentals/issues/256) for someone that might be interested in taking ownership of redoing the chapter on mobile development with Backbone/TodoMVC app using jQuery mobile (that works around common issues like routing) would be really great to have in the book. 

Basically, a tutorial for the app that highlights some of the challenges of getting both working together a little like our existing tutorials. Would you be interested in that? You'd get a credit at the start of the chapter in both the online and physical version of the book.

**Comment by raDiesle** [comment]
Hi Addi,

I once suggested to write the chapter about jQuery Mobile, but it was not clear at that moment,
if it should be part of Backbone fundamentals or not,
so I paused it.

I'll take the responsibility to update & rewrite the chapter for jQuery Mobile.

Greetings

**Comment by addyosmani** [comment]
That is really awesome. Thank you!
